caikit_nlp/modules/text_generation/fine_tuning.py

```diff
@@ -282,28 +278,14 @@ def _preprocess_function(
     ):
         """Pre-process each example to get it prepared for training."""
 
-        # FIXME: Below is currently configured for Seq2Seq only
-
-        def _tokenization_func(
-            example: GenerationTrainRecord,
-        ):
-            model_inputs = tokenizer(
-                example.input,
-                max_length=max_source_length,
-                truncation=True,
-            )
-
-            labels = tokenizer(
-                example.output,
-                max_length=max_target_length,
-                padding="max_length",
-                truncation=True,
-            )
-
-            model_inputs["labels"] = labels["input_ids"]
-
-            return model_inputs
-
-        return SimpleIterableStreamWrapper(
-            train_stream.map(_tokenization_func), shuffle=shuffle
+        (
+            tokenize_function,
+            requires_unwrapping,
+        ) = base_model.build_task_tokenize_function(
+            tokenizer, max_source_length, max_target_length, verbalizer=""
```

Comment:
I think this is a bug - verbalizer template should not be empty here, right? I.e., i.e., we need to specify the actual attr/text to use, otherwise things won't work correctly, since verbalizer template rendering requires us to explicitly specify where the source text is rendered out of too

Reply:
actually, idea here was to make verbalizer optional, since it won't be configured for fine-tuning currently.